src/utils/bundleFeedUtils.ts

```diff
@@ -60,9 +60,12 @@ export namespace bundleFeedUtils {
         return (!template.isHttpTrigger && !template.isTimerTrigger) || bundleTemplateTypes.some(t => isTemplateOfType(template, t));
     }
 
-    export async function getLatestVersionRange(context: IActionContext): Promise<string> {
-        const feed: IBundleFeed = await getBundleFeed(context, undefined);
-        return feed.defaultVersionRange;
+    export async function getLatestVersionRange(_context: IActionContext): Promise<string> {
+        // const feed: IBundleFeed = await getBundleFeed(context, undefined);
+        // return feed.defaultVersionRange;
```

Comment:
Looks like line 45 is also referencing `feed.defaultVersionRange`, which I'm guessing is why the tests are still failing

Reply:
🙈